clang/lib/Driver/ToolChains/Gnu.cpp

```diff
@@ -3060,6 +3060,9 @@ Generic_GCC::Generic_GCC(const Driver &D, const llvm::Triple &Triple,
     : ToolChain(D, Triple, Args), GCCInstallation(D),
       CudaInstallation(D, Triple, Args), RocmInstallation(D, Triple, Args) {
   getProgramPaths().push_back(getDriver().Dir);
+  llvm::Triple TargetTriple(D.getTargetTriple());
+  if (!TargetTriple.isOSWindows())
+    RocmInstallation->init();
```

Comment:
The RocmInstallation should handle Windows, even if it fails, rather than just masking it here and elsewhere.